src/Api/ProjectsApi.php

```diff
@@ -43,8 +81,13 @@ public function projectsMostDownloadedGet($maxVersion = null, $limit = 20, $offs
    */
   public function projectsMostViewedGet($maxVersion = null, $limit = 20, $offset = 0, Flavor $flavor = null, &$responseCode, array &$responseHeaders)
   {
-    $responseCode = Response::HTTP_NOT_IMPLEMENTED;
-    // TODO: Implement projectsMostViewedGet() method.
+    if($maxVersion == null)
```

Comment:
get rid of this check, and setter and just change the default param from null to "0"